addons/packages/ako-operator/1.7.0/bundle/config/overlays/overlay-akodeploymentconfig.yaml

```diff
@@ -0,0 +1,119 @@
+#@ load("/values.star", "values")
+#@ load("@ytt:overlay", "overlay")
+#@ load("@ytt:json", "json")
+
+#@overlay/match by=overlay.subset({"kind": "AKODeploymentConfig", "metadata": {"name": "install-ako-for-all"}})
+---
+#@overlay/replace
+apiVersion: networking.tkg.tanzu.vmware.com/v1alpha1
+kind: AKODeploymentConfig
+metadata:
+    name: install-ako-for-all
+spec:
+#@ if values.akoOperator.config.avi_labels != '""':
+    #@overlay/match missing_ok=True
+    clusterSelector:
+        matchLabels: #@ json.decode(values.akoOperator.config.avi_labels)
+#@ end
+#@ if values.akoOperator.config.avi_controller_version != "":
+    #@overlay/match missing_ok=True
+    controllerVersion: #@ values.akoOperator.config.avi_controller_version
+#@ end
+    cloudName: #@ values.akoOperator.config.avi_cloud_name
+    serviceEngineGroup: #@ values.akoOperator.config.avi_service_engine_group
+    controller: #@ values.akoOperator.config.avi_controller
+    adminCredentialRef:
+        name: #@ values.akoOperator.config.avi_admin_credential_name
+        namespace: #@ values.akoOperator.namespace
+    certificateAuthorityRef:
+        name: #@ values.akoOperator.config.avi_ca_name
+        namespace: #@ values.akoOperator.namespace
+    dataNetwork:
+        name: #@ values.akoOperator.config.avi_data_network
+        cidr: #@ values.akoOperator.config.avi_data_network_cidr
+    controlPlaneNetwork:
+        name: #@ values.akoOperator.config.avi_control_plane_network
+        cidr: #@ values.akoOperator.config.avi_control_plane_network_cidr
+    extraConfigs:
+#@ if values.akoOperator.config.avi_cni_plugin:
```

Comment:
This need to include the new Variables defined in AKOO?
Related to this https://github.com/vmware-tanzu/load-balancer-operator-for-kubernetes/pull/118
Or there will be a follow up PR

Reply:
I guess we need to get the IP Family info from t-f, others does not need to be included .